### PR TITLE
Update documentation & dependencies

### PR DESCRIPTION
**Pull request by ChristianMP**
- Add JSDoc to majority of functions
- Update project name in `package.json`
- Update the README
- Update GTS version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/dde35cf9-8199-4b45-a4c0-f99fe0e1ae9c/deploy-status)](https://app.netlify.com/sites/sg-foodbot/deploys)
+[![Code Style: Google](https://img.shields.io/badge/code%20style-google-blueviolet.svg)](https://github.com/google/gts)
 
 # foodbot
+
+foodbot is a bot for reading ISS food menues, translating the danish menu text and posting it into Slack channels using pretty styling with icons and GIFs.
```
Comment (danchr) [inline README.md]:
```suggestion
A bot for reading ISS food menus, translating the danish menu text and posting it into Slack channels using pretty styling with icons and GIFs.
```
Repetitive and spellbug. Also, lines are _quite_ long in general; intentional? 🤔

File: README.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+
+![Five minute warning](/resource/five_min_warning.png)
+
+### Weekplan
+
+![Weekplan](/resource/weekplan.png)
+
+## GIFs
+
+The varies GIFs used by the bot is stores in a markdown file in the resource folder. This file has a variety of GIFs listed under different sections indicating the type of protein or lack thereof.
```
Comment (danchr) [inline README.md]: Wrong word and grammar.
```suggestion
The various GIFs used by the bot are stored in a markdown file in the resource folder. This file has a variety of GIFs listed under different sections indicating the type of protein or lack thereof.
```
Also, I was once told by a native American speaker that the passive voice appears “weak” in English. As Danes, we tend to use it quite a lot.

File: src/slack.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ import {Block, KnownBlock, WebClient} from '@slack/web-api';
 
 const web = new WebClient(process.env.SLACK_TOKEN);
 
+/** Gets the conversation IDs the bot affiliated with */
```
Comment (danchr) [inline src/slack.ts:5]: Use imperative:
```suggestion
/** Get the conversation IDs the bot affiliated with */
```

File: src/slack.ts [hunk 4]
```diff
@@ -19,6 +20,12 @@ export async function getConversations(): Promise<string[]> {
   return ids;
 }
 
+/**
+ * Publishes a message to the given conversation ID.
```
Comment (danchr) [inline src/slack.ts:24]: Imperative:
```suggestion
 * Publish a message to the given conversation ID.
```

File: src/util.ts [hunk 5]
```diff
@@ -2,6 +2,11 @@ import axios from 'axios';
 import {readFileSync} from 'fs';
 import {v4 as uuidv4} from 'uuid';
 
+/**
+ * Converts the PNG icon filename to a Slack emoji.
```
Comment (danchr) [inline src/util.ts:6]: Ditto:
```suggestion
 * Convert the PNG icon filename to a Slack emoji.
```

File: src/util.ts [hunk 6]
```diff
@@ -26,6 +31,7 @@ export function convertIcon(iconText: string): string {
   }
 }
 
+/** Translates the given text from Danish to English. */
```
Comment (danchr) [inline src/util.ts:36]:
```suggestion
/** Translate the given text from Danish to English. */
```

File: src/util.ts [hunk 7]
```diff
@@ -70,6 +76,12 @@ export async function translateDaToEn(text: string) {
   return result;
 }
 
+/**
+ * Converts the menu item to a menu object with converted icon and translated
```
Comment (danchr) [inline src/util.ts:82]:
```suggestion
 * Convert the menu item to a menu object with converted icon and translated
```

File: src/util.ts [hunk 8]
```diff
@@ -91,6 +103,11 @@ export async function convertToMenuObj(text: string) {
   };
 }
 
+/**
+ * Returns a random gif link from the giphys.md file based on the Slack emoji.
```
Comment (danchr) [inline src/util.ts:109]:
```suggestion
 * Obtain a random gif link from the `giphys.md` file based on the Slack emoji.
```

**Review by danchr — COMMENTED** [review]
A few typos/grammitical errors and some wording suggestions.

File: src/slack.ts (hunk 3 above)
Comment (ChristianMP) [reply src/slack.ts:5]: https://developers.google.com/style/reference-verbs

Also the preferred verb form in our Java code base across departments.

_Edit: I actually think Python at some point said to use imperative, which may be why you suggested it ;D_

Comment (danchr) [reply src/slack.ts:5]: [Busted.](https://peps.python.org/pep-0257/) 😳

File: README.md [hunk 9]
```diff
@@ -1,3 +1,40 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/dde35cf9-8199-4b45-a4c0-f99fe0e1ae9c/deploy-status)](https://app.netlify.com/sites/sg-foodbot/deploys)
+[![Code Style: Google](https://img.shields.io/badge/code%20style-google-blueviolet.svg)](https://github.com/google/gts)
 
 # foodbot
+
+A bot for reading ISS food menus, translating the danish menu text and posting it into Slack channels using pretty styling with icons and GIFs.
```
Comment (danchr) [inline README.md]: Spellbug, missed this earlier:

```suggestion
A bot for reading ISS food menus, translating the Danish menu text and posting it into Slack channels using pretty styling with icons and GIFs.
```

File: README.md [hunk 10]
```diff
@@ -1,3 +1,40 @@
...
+
+## Announcement types
+
+The bot will do three different types of announcements
```
Comment (danchr) [inline README.md]: Missing terminating punctuation, e.g. `:`?
```suggestion
The bot will do three different types of announcements:
```

**Review by danchr — DISMISSED** [review]
Noticed a few minor grammatical issues when re-reading; other than that, looks fine!

(I'd suggest the “Update documentation & dependencies” as a title.)

**Pull request merged**